docs/tutorial/force_calibration.rst

```diff
@@ -1,380 +1,375 @@
 
-Force calibration is used to convert the raw voltages that Bluelake records to actual forces.
-In this tutorial, we will be redoing a calibration that we already performed in Bluelake, to provide an example of using the force calibration module::
-
-    f = lk.File("calibration.h5")
-
-In this tutorial, we want to reproduce calibration item 1.
-We grab the chunk of data that was used for the calibration last time::
-
-    start = f.force1x.calibration[1]["Start time (ns)"]
-    stop = f.force1x.calibration[1]["Stop time (ns)"]
-    force_slice = f.force1x[start:stop]
-
-To be able to calibrate our forces, we first convert back our data to raw voltages.
-We can do this using the previous calibration performed in Bluelake (in this case, the first one in the file)::
-
-    offset = f.force1x.calibration[0]["Offset (pN)"]
-    response = f.force1x.calibration[0]["Rf (pN/V)"]
-    volts = (force_slice - offset) / response
-
-Force calibration models are fit to power spectra. To compute a power spectrum from our data::
-
-    power_spectrum = lk.calculate_power_spectrum(volts.data, sample_rate=volts.sample_rate)
+Force calibration refers to computing the conversion factors needed to convert the raw voltages recorded by the position sensitive detectors to actual forces and displacements.
+The values we wish to calculate are:
 
-This function returns a power_spectrum which we can plot::
+- Trap stiffness :math:`\kappa`, which tells us how stiff the trap is.
+- Force response :math:`R_d`, which converts from voltage to force.
+- Distance response :math:`R_f`, which converts from voltage to distance.
 
-    power_spectrum.plot()
+This tutorial will focus on performing force calibration with Pylake.
+For the theoretical background on the physics of force calibration, please refer to :doc:`theory section on force calibration</theory/force_calibration/force_calibration>`.
 
-.. image:: figures/force_calibration/force_calibration_blocked_spectrum.png
+Undoing the previous calibration
+--------------------------------
 
-Note that the computation of the power spectrum involves some downsampling.
+This tutorial covers calibration using Pylake.
+Our starting point will be data acquired with Bluelake, which is already calibrated.
+Therefore, the first step is to undo the calibration applied by Bluelake to get the raw voltage signals.
+Let's load the dataset::
 
-Additional parameters can be specified to change the amount of downsampling applied (`num_points_per_block`), the range over which to compute the spectrum (`fit_range`) or to exclude specific frequency ranges from the spectrum (`excluded_ranges`)::
+    f = lk.File("passive_calibration.h5")
 
-    power_spectrum = lk.calculate_power_spectrum(volts.data, sample_rate=volts.sample_rate, fit_range=(1e2, 23e3), num_points_per_block=2000, excluded_ranges=[(700, 800), (14500, 14600)])
+We grab the chunk of data that was used for calibration last time::
```

Comment:
I don't think this is correct actually. It contains only one calibration section. The zeroth is always the calibration that was active prior to the slice (so that the voltages can be recovered). There is only one actual calibration here.

Still, I think making this more explicit like you have is good. So how about the following?

```
The force timeline in this file contains a section used for calibration.
We can see calibrations relevant to this file by inspecting the `calibration` attribute for the entire force :class:`~lumicks.pylake.channel.Slice`.
This is a list of dictionaries containing all of the relevant calibration parameters::

    print(len(f.force1x.calibration))
    print(f.force1x.calibration[1])

The first calibration always refers to the calibration that was active at the start of the slice (usually from a calibration that occurred before the start of the slice).
Our goal here is to recalibrate the data in the calibration section (corresponding to `calibrations[1]`).
First we grab the chunk of data that was used for that calibration in Bluelake::
```